src/db_options.rs

```diff
@@ -399,39 +399,6 @@ impl Options {
         }
     }
 
-    /// Hints to the OS that it should not buffer disk I/O. Enabling this
-    /// parameter may improve performance but increases pressure on the
-    /// system cache.
-    ///
-    /// The exact behavior of this parameter is platform dependent.
-    ///
-    /// On POSIX systems, after RocksDB reads data from disk it will
-    /// mark the pages as "unneeded". The operating system may - or may not
-    /// - evict these pages from memory, reducing pressure on the system
-    /// cache. If the disk block is requested again this can result in
-    /// additional disk I/O.
-    ///
-    /// On WINDOWS systems, files will be opened in "unbuffered I/O" mode
-    /// which means that data read from the disk will not be cached or
-    /// bufferized. The hardware buffer of the devices may however still
-    /// be used. Memory mapped files are not impacted by this parameter.
-    ///
-    /// Default: true
-    ///
-    /// # Example
-    ///
-    /// ```
-    /// use rocksdb::Options;
-    ///
-    /// let mut opts = Options::default();
-    /// opts.set_allow_os_buffer(false);
-    /// ```
-    #[deprecated(since="0.7.0", note="replaced with set_use_direct_reads/set_use_direct_writes methods")]
-    pub fn set_allow_os_buffer(&mut self, is_allow: bool) {
```

Comment:
Aah got you, the merger of the cherry-pick must have destroyed it - I'll look at your commit.